GTG/gtk/editor/editor.py

```diff
@@ -572,18 +573,38 @@ def inserttag_clicked(self, widget):
 
     def on_parent_select(self, widget):
         parents = self.task.get_parents()
+
         if len(parents) == 1:
-            self.vmanager.open_task(parents[0])
-        elif len(self.task.get_parents()) > 1:
-            '''
-            TODO (jakubbrindza): If there is more than 1 parent,open a popover
-            with their names and allow choosing which one to open.
-            '''
-            pass
+            self.open_parent_clicked(None, parents[0])
+        elif len(parents) > 1:
+            self.show_multiple_parent_popover(parents)
+
+    def show_multiple_parent_popover(self, parent_ids):
+        parent_box = Gtk.Box.new(Gtk.Orientation.VERTICAL, 0)
+        for parent in parent_ids:
+            parent_name = self.req.get_task(parent).get_title()
+            button = Gtk.ToolButton.new(None, parent_name)
+            button.connect("clicked", self.open_parent_clicked, parent)
+            parent_box.add(button)
+
+        self.parent_popover = Gtk.Popover.new(self.parent_button)
+        self.parent_popover.add(parent_box)
+        self.parent_popover.set_property("border-width", 0)
+        self.parent_popover.set_position(Gtk.PositionType.BOTTOM)
+        self.parent_popover.set_transitions_enabled(True)
+        self.parent_popover.show_all()
 
     # On click handler for open_parent_button's menu items
-    def open_parent(self, widget, tid):
-        self.vmanager.open_task(tid)
+    def open_parent_clicked(self, widget, parent_id):
```

Comment:
Please, rename this to `on_parent_item_clicked` (follow the convention that signal callbacks are on_something)